src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/commissioner/CommissioningFragment.java

```diff
@@ -106,8 +118,7 @@ public void onClick(View view) {
 
   @Override
   public void onChanged(@Nullable WorkInfo workInfo) {
-    TextView statusText = getActivity().findViewById(R.id.status_text);
-    ProgressBar progressBar = getActivity().findViewById(R.id.commissioning_progress);
+    progressBar = getActivity().findViewById(R.id.commissioning_progress);
```

Comment:
You can remove lines 121, 136, and 141 since those views have been assigned in onViewCreated(..) already.

Reply:
removed.